.ci/setup_env_centos.sh

```diff
@@ -88,3 +89,11 @@ chronic sudo -E yum install -y procenv
 
 echo "Install haveged"
 chronic sudo -E yum install -y haveged
+
+if [ "$KATA_KSM_THROTTLER" == "yes" ]; then
+	package="kata-ksm-throttler"
+	echo "Install ${package}"
+	chronic sudo -E yum install ${package}
```

Comment:
Thinking aloud: it might be useful if we created an `install_packages()` function in `.ci/lib.sh` to wrap all the `dnf`/`yum`/`apt` stuff. It might even be worth adding functions like `enable_services()` / `start_services()` to allow us to simply handle things like logging, std stream redirection.

None of this required for this PR though :smile: